docs/tasks.rst

```diff
@@ -231,6 +231,111 @@ Options
 
 	 Metadata API version to use, if not project__package__api_version.
 
+**assign_compact_layout**
+==========================================
+
+**Description:** Assigns the Compact Layout specified in the 'value' option to the Custom Objects in 'api_names' option.
+
+**Class:** cumulusci.tasks.metadata_etl.UpdateMetadataFirstChildTextTask
+
+Metadata ETL task to update a single child element's text within metadata XML.
```

Comment:
@davisagli  I suggested to @spelak-salesforce that we publish a preconfigured task in `cumulusci.yml` for the known concrete use of this generic task. That unfortunately results in reduplicating the generic tasks's docs. 

Is there a better solution? I suppose we could ship a separate Python task but part of the virtue of this PR is not requiring additional Python.

Reply:
@davidmreed as of right now we need a separate Python class in order to provide different task_docs. It would be a small tweak of the doc_task function to allow for overriding task_docs in the YAML task_config

Reply:
I think having the task available outweighs the minor confusion in the task reference. Happy to put that tweak on my backlog.